plugins/ratio/ratio.php

```diff
@@ -17,28 +17,45 @@ class rRatio
 	public $hash = "ratio.dat";
 	public $rat = array();
 	public $default = 0;
+	private $version = 3.10;
 
 	static public function load()
 	{
 		$cache = new rCache();
 		$rt = new rRatio();
 		if(!$cache->get($rt))
+		{
 			$rt->fillArray();
+			$rt->version = 3.11;
+		}
+		elseif ($rt->version != 3.11)
+		{
+			$rt->migrate();
+			$rt->pad();
+			$rt->version = 3.11;
+		}
 		else
 			$rt->pad();
 		return($rt);
 	}
+	private function migrate()
+	{
+		for($i=0; $i<count($this->rat); $i++)
+		{
+			$this->rat[$i]["upload"] /= 1000;
```

Comment:
1024 maybe?

Reply:
For example, we had 1024 in the upload.
In the past we sent to rtorrent value 1024 * 1024 * 1024 = 1073741824
Now we will send 1024 / 1000 * 1024 * 1024 * 1024 = 1099511630

Reply:
Okay, I fixed it.